### DISCUSSION TITLE
Vixen (NSFW)

### DISCUSSION THREAD

**Discussion by ovidiuux** [iptv-org/community#857] [Show and tell]
### Channel ID

VixenTV.ca

### Stream URL

https://www.hlsrundle-stream-iptv.gq/api/hls/vixen.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Wonder what the source is for this one... lol.

**Comment by ovidiuux** [comment]
> Wonder what the source is for this one... lol.

No, actually yes. When VIXEN is no longer available, a stream from Adult IPTV is put on. it's not 24/7, but it recovers soon

**Comment by BellezaEmporium** [comment]
> > Wonder what the source is for this one... lol.
> 
> No, actually yes. When VIXEN is no longer available, a stream from Adult IPTV is put on. it's not 24/7, but it recovers soon

I mean, your server your streams lol. But if the base is from an xtream-codes server, it's quite difficult to keep them unless it's not some credentials found online.

**Comment by freearhey** [comment]
https://www.hlsrundle-stream-iptv.gq/api/hls/vixen.m3u8 (offline)
<img width="403" alt="image" src="https://user-images.githubusercontent.com/7253922/232167572-abc9eed6-50fc-4db4-8a68-498b0d6d1336.png">

**Comment by freearhey** [comment]
http://clientportal.link:80/6yjrsES6RO/iRWHJn390/60033

**Comment by BellezaEmporium** [comment]
> http://clientportal.link:80/6yjrsES6RO/iRWHJn390/60033

I am not quite OK with using Xtream-Codes servers...

**Comment by freearhey** [comment]
> As of January 30th, 2024, we have stopped distributing NSFW channels. For more information, please look at https://github.com/iptv-org/iptv/issues/15723.

https://github.com/iptv-org/iptv?tab=readme-ov-file#playlists
